Mathlib/Combinatorics/SimpleGraph/Operations.lean

```diff
@@ -5,6 +5,7 @@ Authors: Jeremy Tan
 -/
 import Mathlib.Combinatorics.SimpleGraph.Finite
 import Mathlib.Combinatorics.SimpleGraph.Maps
+import Mathlib.Combinatorics.SimpleGraph.Subgraph
```

Comment:
Is this unavoidable?

Reply:
Generally these files could probably do with a reorganization. e.g. the first declaration in this file seems to have nothing to do with the module doc, and I suspect is only here because it's the first place the imports are available.

Reply:
I agree that `card_edgeFinset_eq` is a little bit in an odd spot here.

For what I'm doing, the alternatives I see are:
- Move edge to `Basic.lean`
- Move result to another file without changing imports (Clique is the only file importing this)
- Add this to another file that already has Subgraph and add the Operations import 
- Create a new file

My intuition is that adding the Subgraph import on this file makes the most sense, but I'm happy to be convinced otherwise.